src/map/status.cpp

```diff
@@ -8742,71 +8746,104 @@ t_tick status_get_sc_def(struct block_list *src, struct block_list *bl, enum sc_
 				tick>>=1;
 				tick_def = (status->vit*200)/3;
 			}
+#else
+			sc_def = status->vit * 100 - levelAdv;
+			tick_def2 = -2000;
 #endif
 			break;
 		case SC_STUN:
+#ifndef RENEWAL
 			sc_def = status->vit*100;
 			sc_def2 = status->luk*10 + status_get_lv(bl)*10 - status_get_lv(src)*10;
 			tick_def2 = status->luk*10;
+#else
+			sc_def = status->vit * 100 - levelAdv;
+			tick_def2 = -500;
+#endif
 			break;
 		case SC_SILENCE:
 #ifndef RENEWAL
 			sc_def = status->vit*100;
 			sc_def2 = status->luk*10 + status_get_lv(bl)*10 - status_get_lv(src)*10;
+			tick_def2 = status->luk*10;
 #else
-			sc_def = status->int_*100;
-			sc_def2 = (status->vit + status->luk) * 5 + status_get_lv(bl)*10 - status_get_lv(src)*10;
+			sc_def = status->int_ * 100 - levelAdv;
+			tick_def2 = -2000;
 #endif
-			tick_def2 = status->luk*10;
 			break;
 		case SC_BLEEDING:
 #ifndef RENEWAL
 			sc_def = status->vit*100;
 			sc_def2 = status->luk*10 + status_get_lv(bl)*10 - status_get_lv(src)*10;
+			tick_def2 = status->luk*10;
 #else
-			sc_def = status->agi*100;
-			sc_def2 = status->luk*10 + status_get_lv(bl)*10 - status_get_lv(src)*10;
+			sc_def = status->agi * 100 - levelAdv;
+			tick_def2 = -12000;
 #endif
-			tick_def2 = status->luk*10;
 			break;
 		case SC_SLEEP:
 #ifndef RENEWAL
 			sc_def = status->int_*100;
 			sc_def2 = status->luk*10 + status_get_lv(bl)*10 - status_get_lv(src)*10;
+			tick_def2 = status->luk*10;
 #else
-			sc_def = status->agi*100;
-			sc_def2 = (status->int_ + status->luk) * 5 + status_get_lv(bl)*10 - status_get_lv(src)*10;
+			sc_def = status->agi * 100 - levelAdv;
+			tick_def2 = -2000;
 #endif
-			tick_def2 = status->luk*10;
 			break;
 		case SC_STONE:
+#ifndef RENEWAL
 			sc_def = status->mdef*100;
 			sc_def2 = status->luk*10 + status_get_lv(bl)*10 - status_get_lv(src)*10;
 			tick_def = 0; // No duration reduction
+#else
+			sc_def = status->mdef * 100 - levelAdv;
+			tick_def2 = -3000;
+#endif
 			break;
 		case SC_FREEZE:
+#ifndef RENEWAL
 			sc_def = status->mdef*100;
 			sc_def2 = status->luk*10 + status_get_lv(bl)*10 - status_get_lv(src)*10;
 			tick_def2 = status_src->luk*-10; // Caster can increase final duration with luk
+#else
+			sc_def = status->mdef * 100 - levelAdv;
+			tick_def2 = -3000;
```

Comment:
Just as reminder (already talked with you about it on Discord): When I implementated the official status change durations in skill_cast_db, I included those times in the variable casting time as I set those times based on my tests on official servers. Now that we properly implemented it as fixed duration here, we need to remove those added times from the skill_db.yml files.

80-90% of all skills just use the default duration, but those default durations are not implemented yet.

The remaining skills are usually those that have a different durations per level, such a Frost Diver, which has a variable duration of 3000ms*level. I currently put as 2500ms+3000ms*level as duration in the file (the times I measured during my tests were off by 500ms, probably due to latency). But now we add those 3000ms here, so the duration in the skill_db.yml should not include it.

So either we comment the fixed duration out in this commit for now, or we update the skill_db.yml at the same time.